BSP/STM32F413/Src/BSP_SPI.c

```diff
@@ -16,6 +20,8 @@ static uint8_t SPI_WriteRead(uint8_t txData){
 	return SPI1->DR & 0x00FF;
 }
 
+
+#ifdef BAREMETAL
```

Comment:
With the way this is set up. If we compile with RTOS, then the SPI_Init function will not be compiled.